server/channels/db/migrations/mysql/000114_user_reporting_changes.down.sql

```diff
@@ -0,0 +1,14 @@
+SET @preparedStatement = (SELECT IF(
```

Comment:
Why aren't we using the props column for this? I thought that's what we planned to do?

Reply:
I had noticed that we weren't using the `Props` column for really anything at all, at least nothing related to the core app. Since this is a core app feature I thought it best to add another column.

Let me know if this is going to be an issue.

Reply:
Maybe not for this table, but we do use props for other tables, so I don't foresee any issue.